src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/embeddedGrammars/html.tmLanguage.json

```diff
@@ -108,7 +108,7 @@
 													"name": "punctuation.definition.string.end.html"
 												},
 												"1": {
-													"name": "source.css"
+													"name": "source.css-ignored-vscode"
```

Comment:
Does anyone know what this means? Is this bad?

Reply:
Looks like this is something that vscode specifically does for two specific types.. I wonder if we should filter it out, to not cause any problems/differences?
https://cs.github.com/microsoft/vscode/blob/ecb3421521db4764becfd198a79ca88b6aad73c2/extensions/html/build/update-grammar.mjs?q="ignored-vscode"#L13-L18

@NTaylorMullen do you have any context on this?

Reply:
Looked more into this, seems like the actual non-test tmLanguage file already had these, so this presumably isn't a problem, its just the test files weren't updated previously.